crates/iceberg-catalog/src/catalog/io.rs

```diff
@@ -331,4 +488,19 @@ mod tests {
             test_remove_all(cred, profile).await;
         }
     }
+
+    #[tokio::test]
+    #[serial]
+    async fn test_remove_all_hdfs() {
+        let (dfs, prof, creds) = hdfs::test::test_profile();
+        test_remove_all(creds, StorageProfile::Hdfs(prof)).await
+    }
```

Comment:
Should be removed since we dont want to depend on java runtime plus serial is a no go for us